src/scripts/update-icons.ts

```diff
@@ -88,12 +88,12 @@ const mutateIcon = (iconName) => {
   if (!isProbablyColor(iconName)) {
     const idealColor = '#62757E'
     for (const filled of document.querySelectorAll('[fill]')) {
-      if (isAllowedColor(filled.getAttribute('fill'))) continue
+      if (isAllowedColor(filled.getAttribute('fill')!)) continue
```

Comment:
Out of curiosity, what is the `!` operator doing here?

Reply:
It means "this is not null/undefined"